.github/workflows/pull-request-teardown.yml

```diff
@@ -24,15 +24,21 @@ jobs:
         RENKUBOT_KUBECONFIG: ${{ secrets.RENKUBOT_KUBECONFIG }}
       with:
         kubectl: 1.16.2
-        helm: 2.16.1
+        helm: 3.3.1
         command: |
           export PR_NUMBER=${{ github.event.number }}
           echo $PR_NUMBER
           export RENKU_RELEASE="ci-${PR_NUMBER}-renku"
-          helm init --client-only
+          export RENKU_NAMESPACE="ci-${PR_NUMBER}-renku"
           echo "$RENKUBOT_KUBECONFIG" > renkubot-kube.config
-          helm delete --purge $RENKU_RELEASE
-          helm delete --purge $RENKU_RELEASE-tmp-notebooks || true
+          helm uninstall $RENKU_RELEASE --namespace $RENKU_NAMESPACE
+          kubectl delete all --all --namespace $RENKU_NAMESPACE
```

Comment:
I think you could simplify this and just delete the namespace without previously deleting the resources in it.

Reply:
I would do what you propose but this is a bit tricky actually. Yesterday I've been hitting some issues when a namespace was stuck in `Terminating` state and had to manually delete them. Also, the first time I tried the teardown (that had the deletion of a namespace) not all of the resources were deleted not really sure why. I prefer better to be on the safe side for now.

Reply:
ok, if you've experienced this to work better like that, let's leave it.